unitpackage/collection.py

```diff
@@ -251,3 +251,20 @@ def __getitem__(self, identifier):
                 f"The collection has more than one entry with identifier '{identifier}'."
             )
         return entries[0]
+
+    def save_entries(self, outdir=None):
+        r"""
+        Saves all entries from a Collection as datapackages (CSV and JSON)
+        to a specific output directory :param outdir:.
+
+        EXAMPLES::
+
+            >>> db = Collection.create_example()
+            >>> db.save_entries(outdir='test/generated/saved_collection/')
+            >>> import glob
+            >>> glob.glob('test/generated/saved_collection/**.json') # doctest: +NORMALIZE_WHITESPACE
+            ['test...
```

Comment:
For all the tests where files are generated, we should adopt an approach used in `svgdigitizer`, where we compare the content of the output files to that of the expected files.